src/js/entities-service/entities/comments.js

```diff
@@ -8,24 +8,26 @@ const TYPE = 'comments';
 
 const _Model = BaseModel.extend({
   type: TYPE,
-// TODO:
-//  urlRoot() {
-//    if (this.isNew()) return `/api/actions/${ this.get('_action') }/relationships/comments`;
-//
-//    return '/api/comments';
-//  },
-//  getClinician() {
-//    return Radio.request('entities', 'clinicians:model', this.get('_clinician'));
-//  },
-//  saveAll(attrs) {
-//    attrs = _.extend({}, this.attributes, attrs);
-//
-//    const relationships = {
-//      clinician: this.toRelation(attrs._clinician),
-//    };
-//
-//    return this.save(attrs, { relationships }, { wait: true });
-//  },
+  urlRoot() {
+    if (this.isNew()) return `/api/actions/${ this.get('_action') }/relationships/comments`;
+
+    return '/api/comments';
+  },
+  validate({ body }) {
```

Comment:
I believe the body of a comment is `message`

Reply:
Ah thanks, meant to ask about that.